capstone/src/test/java/com/google/sps/data/DataStoreTest.java

```diff
@@ -34,14 +41,99 @@ private void doTest() {
     assertEquals(2, ds.prepare(new Query("test")).countEntities(withLimit(10)));
   }
 
-  /* Commented these out since tests fail, will add more tests in future PRS */
-  //   @Test
-  //   public void testInsert1() {
-  //     doTest();
-  //   }
+  @Test
+  public void testStateleakInsert1() {
+    doTest();
+  }
+
+  @Test
+  public void testStateleakInsert2() {
+    doTest();
+  }
+
+  public boolean inLocalStore(Entity entity) {
+    try {
+      datastore.get(entity.getKey());
+    } catch (EntityNotFoundException e) {
+      return false;
+    }
+
+    return true;
+  }
+
+  /**
+   * Creates an entity based on given survey response values
+   *
+   * @param kind Namespace in datastore
+   * @param zipCode Location of where survey was taken
+   * @param id Unique id given to each survey
+   * @param score Sentiment score for the survey response
+   * @return Entity data store object representing a survey response
+   */
+  public Entity getMockEntity(String kind, String zipCode, int id, float score) {
+    Entity entity = new Entity(kind, id);
+    entity.setProperty("zipCode", zipCode);
+    entity.setProperty("score", score);
 
-  //   @Test
-  //   public void testInsert2() {
-  //     doTest();
-  //   }
+    return entity;
+  }
+
+  @Test
+  public void testMockEntityStorage() throws IOException {
+    ArrayList<Entity> testEntities = new ArrayList<Entity>();
+    Entity a = getMockEntity("ResponseTest", "94101", 1, 0.5f);
+    Entity b = getMockEntity("ResponseTest", "94151", 2, 0.83f);
+
+    testEntities.add(a);
+    testEntities.add(b);
+
+    datastore.put(testEntities);
+
+    Entity entity = new Entity("ResponseTest", 2);
+
+    assertTrue(inLocalStore(entity));
```

Comment:
For all of the asserts in this file -- Junit has an option that allows you to also add a string as an error message if the asserts fail, which can be useful for debugging if tests do not pass.